users/models.py

```diff
@@ -131,3 +131,26 @@ def save(self, **kwargs):
             self.last_vote_affirmation = timezone.now()
 
         return super().save(**kwargs)
+
+
+class UserGroup(models.Model):
+    name = models.CharField(max_length=255)
+    location = models.CharField(max_length=255)
+    url = models.URLField('URL')
+
+    TYPE_MEETUP = 'meetup'
+    TYPE_DISTRIBUTION_LIST = 'distribution list'
+    TYPE_OTHER = 'other'
+
+    TYPE_CHOICES = (
+        (TYPE_MEETUP, 'meetup'),
+        (TYPE_DISTRIBUTION_LIST, 'distribution list'),
+        (TYPE_OTHER, 'other')
+    )
+    url_type = models.CharField(
+        max_length=20,
+        choices=TYPE_CHOICES)
```

Comment:
Style nit: It would be better to use hanging indentation to make future diff clearer when we add a new setting to the field:

```py
url_type = models.CharField(
    max_length=20,
    choices=TYPE_CHOICES,  # And please add a trailing comma.
)
```